component/component/vshn_mariadb.jsonnet

```diff
@@ -0,0 +1,109 @@
+local com = import 'lib/commodore.libjsonnet';
+local kap = import 'lib/kapitan.libjsonnet';
+local kube = import 'lib/kube.libjsonnet';
+
+local comp = import 'lib/appcat-compositions.libsonnet';
+local crossplane = import 'lib/crossplane.libsonnet';
+
+local common = import 'common.libsonnet';
+local xrds = import 'xrds.libsonnet';
+
+local slos = import 'slos.libsonnet';
+
+local inv = kap.inventory();
+local params = inv.parameters.appcat;
+local mariadbParams = params.services.vshn.mariadb;
+
+local serviceNameLabelKey = 'appcat.vshn.io/servicename';
+local serviceNamespaceLabelKey = 'appcat.vshn.io/claim-namespace';
+
+local connectionSecretKeys = [
+  'ca.crt',
+  'tls.crt',
+  'tls.key',
+  'MARIADB_HOST',
+  'MARIADB_PORT',
+  'MARIADB_USERNAME',
+  'MARIADB_PASSWORD',
+  'MARIADB_URL',
+];
+
+local promRuleMariaDBSLA = common.PromRuleSLA(params.services.vshn.mariadb.sla, 'VSHNMariaDB');
+
+local mariadbPlans = common.FilterDisabledParams(mariadbParams.plans);
+
+local xrd = xrds.XRDFromCRD(
+  'xvshnminios.vshn.appcat.vshn.io',
+  xrds.LoadCRD('vshn.appcat.vshn.io_vshnminios.yaml', params.images.appcat.tag),
+  defaultComposition='vshnminio.vshn.appcat.vshn.io',
+  connectionSecretKeys=connectionSecretKeys,
+) + xrds.WithPlanDefaults(mariadbPlans, mariadbParams.defaultPlan);
+
+local composition =
```

Comment:
Why aren't you using your shiny new `vshn_appcat_service()` here?

Reply:
Forgot to remove this file before I started with the vshn_appcat_service()